common/middleware.go

```diff
@@ -453,48 +456,37 @@ var ContentTypeMarshallers map[string]Marshaller = map[string]Marshaller{
 	//	"*/*": jsonMarshaller{},
 }
 
-// Authenticator is the interface that will be used by AuthMiddleware
-// to provide authentication. Details to be worked out later.
-type Authenticator interface {
-	// As this is a placeholder, we are not dealing with
-	// details yet, tokens vs credentials, principals vs roles, etc.
-	Authenticate() Principal
-}
-
-type PlaceholderAuth struct {
-}
-
-func (p PlaceholderAuth) Authenticate() Principal {
-	return Principal{"asdf", []string{"read", "write", "execute"}}
-}
-
-// Principal is a placeholder for a Principal structure
-// for authentication. We'll leave roles and
-// other stuff for later.
-type Principal struct {
-	Id string
-	// This really is a placeholder
-	Rights []string
-}
-
 // AuthMiddleware wrapper for auth.
```

Comment:
Could we add a comment about who uses the public key, where it comes from, why, what, how?